src/indieweb_utils/replies/context.py

```diff
@@ -29,298 +40,316 @@ def get_reply_context(url, twitter_bearer_token=None):
     :rtype: list
     """
 
-    h_entry = None
-    photo_url = None
-    site_supports_webmention = False
+    return_object = ReplyContext
+    photo_url = ""
     parsed_url = url_parse.urlsplit(url)
     http_headers = {"Accept": "text/html", "User-Agent": "indieweb_utils"}
 
-    if url.startswith("https://") or url.startswith("http://"):
-        try:
-            page_content = requests.get(url, timeout=10, verify=False, headers=http_headers)
-        except:
-            return True, {"error": "Page content could not be retrieved."}
+    if not url.startswith("https://") and not url.startswith("http://"):
+        return return_object
```

Comment:
If it's not an http url, we don't support it, right? Rather than return an empty class object, we should raise an error.

```python
raise UnsupportedScheme(f"{parsed_url.scheme} is not supported")
```